src/main/java/com/aws/greengrass/mqtt/bridge/auth/MQTTClientKeyStore.java

```diff
@@ -56,6 +56,9 @@ public class MQTTClientKeyStore {
     @FunctionalInterface
     public interface UpdateListener {
         void onCAUpdate();
+
+        default void onClientCertUpdate() {
```

Comment:
why is this new? We already respond to cert rotations correctly in the mqtt 3 client, so why do we need this new callback, surely we must be handling it already

Reply:
Nevermind. I see

Reply:
for mqtt3, bridge just stores new cert in keystore, and mqtt3 client will just pick it up when it creates the new connection. it doesn't respond automatically to client cert updates currently